src/ResourceManager/MachineLearning/Commands.MachineLearning/Cmdlets/RemoveAzureMLWebService.cs

```diff
@@ -0,0 +1,92 @@
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.MachineLearning.Utilities;
+using Microsoft.Azure.Management.MachineLearning.WebServices.Models;
+using Microsoft.WindowsAzure.Commands.Common;
+
+namespace Microsoft.Azure.Commands.MachineLearning
+{
+    [Cmdlet(VerbsCommon.Remove, WebServicesCmdletBase.CommandletSuffix)]
+    [OutputType(typeof(void))]
+    public class RemoveAzureMLWebService : WebServicesCmdletBase
+    {
+        protected const string RemoveByNameGroupParameterSet = 
+            "Remove an Azure ML web service resouce by name and resource group.";
+        protected const string RemoveByObjectParameterSet = 
+            "Remove an Azure ML web service specified as an object.";
+
+        [Parameter(
+            ParameterSetName = RemoveAzureMLWebService.RemoveByNameGroupParameterSet, 
+            Mandatory = true, 
+            HelpMessage = "The name of the resource group for the Azure ML web service.")]
+        [ValidateNotNullOrEmpty]
+        public string ResourceGroupName { get; set; }
+
+        [Parameter(
+            ParameterSetName = RemoveAzureMLWebService.RemoveByNameGroupParameterSet, 
+            Mandatory = true, 
+            HelpMessage = "The name of the web service.")]
+        [ValidateNotNullOrEmpty]
+        public string Name { get; set; }
+
+        [Parameter(
+            ParameterSetName = RemoveAzureMLWebService.RemoveByObjectParameterSet, 
+            Mandatory = true, 
+            HelpMessage = "The machine learning web service object.", 
+            ValueFromPipeline = true)]
+        [ValidateNotNullOrEmpty]
+        public WebService MlWebService { get; set; }
+
+        [Parameter(Mandatory = false, HelpMessage = "Do not ask for confirmation.")]
+        public SwitchParameter Force { get; set; }
+
+        protected override void RunCmdlet()
+        {
+            if (string.Equals(
+                        this.ParameterSetName, 
+                        RemoveAzureMLWebService.RemoveByObjectParameterSet, 
+                        StringComparison.OrdinalIgnoreCase))
+            {
+                string subscriptionId, resourceGroup, webServiceName;
+                if (!CmdletHelpers.TryParseMlWebServiceMetadataFromResourceId(
+                                    this.MlWebService.Id, 
+                                    out subscriptionId, 
+                                    out resourceGroup, 
+                                    out webServiceName))
+                {
+                    throw new ValidationMetadataException(Resources.InvalidWebServiceIdOnObject);
+                }
+
+                this.ResourceGroupName = resourceGroup;
+                this.Name = webServiceName;
+            }
+
+            this.ConfirmAction(
```

Comment:
This is likely the wrong thing to do (although this pattern is used in many places, it is actually incorrect and is in process of beign changed across the board).

In this case, you probably don't want a -Force parameter, unless there are some special circumstances in which you want an extra prompt for the user (for example, if you can detect that the web service is in use, you might want an extra prompt).

For the no special deletion, just use:
```
if (ShouldProcess(target, action) {...normal execution...}
```
and do not have a force parameter.  If there are special conditions under which you'd like to do an extra prompt:
```
if (ShouldProcess(target, action)) {
... actions
  if (!SpecialConditionPredicate() || Force || ShouldContinue(...)) { ... perform deletion ...}
}
```

LMK if this isn't clear